Grammar/python.gram

```diff
@@ -134,7 +134,8 @@ import_from[stmt_ty]:
         _Py_ImportFrom(NULL, b, _PyPegen_seq_count_dots(a), EXTRA) }
 import_from_targets[asdl_seq*]:
     | '(' a=import_from_as_names [','] ')' { a }
-    | import_from_as_names
+    | import_from_as_names !','
+    | invalid_import_from_targets
```

Comment:
We can still put this one the last, no?

Reply:
Yup. That's what I mean. Sorry, @isidentical, for the bad suggestion. Got tripped up by the Github UI.

Reply:
Fixed it